add-ons/che/README.md

```diff
@@ -25,33 +26,49 @@ The best way of using this add-on is via the [`minishift add-ons apply`](https:/
 
 <a name="install-add-on"></a>
 ### Install add-on
+Che default stacks require root privilege to run sshd. Without root privilege, workspaces will take more time to timeout and start.
+One workaround would be to enable the `anyuid` addon. Another way would be to [replace the existing stacks](#replace-stacks) with the ones without the `sshd` agent.
+
+    $ minishift addons install --defaults
+    $ minishift addons enable anyuid
 
 Clone this repository onto your local machine and then install the add-on via:
 
     $ minishift addons install <path_to_directory_containing_this_readme>
     $ minishift addons enable che
 
 
-`enable` will setup eclipse che at next minishift start
+`enable` will setup eclipse che at next Minishift start
```

Comment:
I think this should be:

will setup Eclipse Che at next `minishift start`

Reply:
Yes, for all CLI operation, it should `minishift`.

Reply:
ok i make that fix